kmip/pie/objects.py

```diff
@@ -342,6 +359,8 @@ def __init__(self, algorithm, length, value, masks=None,
                 type of algorithm for the key.
             length(int): The length in bits of the key.
             value(bytes): The bytes representing the key.
+            format_type(KeyFormatType): An enumeration defining the format of
+                the key value.
```

Comment:
I think we should define what values are acceptable. I think public keys should be raw or X.509 and X.509 is the default. An X.509 means the value is a DER-encoded bytearray for a SubjectPublicKeyInfo ASN.1 structure. A raw type means the public key is a DER-encoded representation of the public key conforming to its ASN.1 spec. All other values should raise an exception.